packages/graphql/lib/get-graphql-error.js

```diff
@@ -0,0 +1,36 @@
+const getHeaders = (status, headers) => {
+  if (status === 429) {
+    const retryAfter = headers.get('retry-after');
+    return retryAfter ? { 'retry-after': retryAfter } : {};
+  }
+  return {};
+};
+
+module.exports = error => {
+  if (!error.networkError || !error.networkError.response) {
+    return error;
+  }
+
+  const { stack, response } = error.networkError;
+  const { status: statusCode, statusText, headers: responseHeaders } = response;
+  const status = statusCode === 429 ? 429 : 500;
+  const headers = getHeaders(status, responseHeaders);
+  const message = statusCode === 200 ? 'Not Acceptable' : statusText;
+  const fetchError = Object.assign(new Error(message), {
+    name: 'HopsGraphQlError',
+    original: error.networkError,
+    headers,
+    status,
+  });
+  const fetchErrorStack = fetchError.stack
```

Comment:
I think it is totally ok to keep the current stack trace. And the stack trace of `networkError` isn't that helpful either because that only shows where the fetch function got called...